src/Fetch.res

```diff
@@ -73,6 +73,7 @@ module Headers = {
 
   @new external empty: t = "Headers"
   @new external make: init => t = "Headers"
+  @new external clone: t => t = "Headers"
```

Comment:
I can't find where this is defined. It doesn't seem to be in the Fetch standard at least, and I don't think it's on the `iterable` protocol either. Do you know?

Reply:
Could you just add a comment marking it as `non-standard` then?